xbmc/addons/RepositoryUpdater.cpp

```diff
@@ -332,6 +332,18 @@ void CRepositoryUpdater::ScheduleUpdate()
               nextCheck.GetAsLocalizedDateTime(), delta / 1000);
   }
 
+  if (scheduleType == UpdateScheduleType::Regular) {
+    // Enforce minimum hold-off time of 1 hour between regular updates - this is especially 
+    // important to handle all sorts of failure cases (e.g., failure to update the add-on database)
+    // that would otherwise lead to an immediate new update attempt and continuous hammering of the servers.
+    delta = std::max(1 * 60 * 60 * 1'000, delta);
```

Comment:
Can we just use std::chrono types?

Reply:
no

Reply:
I'll do another PR for changing the whole file to chrono later, but for omega I want to have it as nonintrusive as possible

Reply:
Boooo 😛